src/Sylius/Behat/Service/NotificationValidator.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Service;
+
+use Sylius\Behat\Service\Accessor\NotificationAccessorInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+final class NotificationValidator implements NotificationValidatorInterface
+{
+    /**
+     * @var NotificationAccessorInterface
+     */
+    private $notificationAccessor;
+
+    /**
+     * @param NotificationAccessorInterface $notificationAccessor
+     */
+    public function __construct(NotificationAccessorInterface $notificationAccessor)
+    {
+        $this->notificationAccessor = $notificationAccessor;
+    }
+
+    /**
+     * @param string $resource
+     */
+    public function checkDeletionNotification($resource)
+    {
+        $this->assertPositiveMessage();
```

Comment:
Probably right now there is no scenario where this will happen, but it is definitely a possible scenario. However I would leave it for now, there will be only one place to change this logic if we will find it buggy.